<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Satyas-MacBook-Pro.local" name="com.qtpselenium.suite.shop.CheckProductDisplay" tests="1" failures="1" timestamp="31 Jan 2016 15:05:28 GMT" time="0.001" errors="0">
  <testcase name="testcaseA1" time="0.001" classname="com.qtpselenium.suite.shop.CheckProductDisplay">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.qtpselenium.util.Xls_Reader.getRowCount(Xls_Reader.java:46)
	at com.qtpselenium.util.TestUtil.isSuiteRunnable(TestUtil.java:10)
	at com.qtpselenium.suite.cart.TestSuiteBase.checkSuiteSkip(TestSuiteBase.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:525)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:202)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:130)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:260)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1035)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.TestNG.privateMain(TestNG.java:1182)
	at org.testng.TestNG.main(TestNG.java:1146)
]]>
    </failure>
  </testcase>
</testsuite>
